code/modules/mob/inventory/_docs.dm

```diff
@@ -0,0 +1,96 @@
+// ITEMS //
+/*
+Most procs from that categoy located in:
+ code/game/objects/items.dm
+*/
+
+
+/obj/item/proc/update_wear_icon(redraw_mob = TRUE)
+/*
+Replacement for outdated update_holding_icon()/update_clothing_icon()
+Found occupied slot and update it icon
+*/
+
+
+/obj/item/proc/pre_equip(var/mob/user, var/slot)
+/*
+Called just before an item is placed in an equipment slot.
+Use this to do any necessary preparations for equipping
+Immediately after this, the equipping will be handled and then equipped will be called.
+Returning a non-zero value will silently abort the equip operation
+Important notes:
+	- Can take few ticks if needed!
+	- Called before unequip (if holding by mob) and can still be located in another mob!
+By default here located equip sound calls.
+*/
+
+
+/obj/item/proc/equipped(var/mob/Mob, var/slot)
+/*
+Called after an item is placed in an equipment slot
+User is mob that equipped it
+Slot uses the slot_X defines found in items_clothing.dm
+*/
+
+/obj/item/proc/dropped(mob/Mob)
+/*
+Called whenever an item is dropped from INVENTORY SLOT.
+Important note:
+	- It is called after loc is set, so if placed in a container its loc will be that container.
+By default drop zoom if enabled.
+*/
+
+
+/obj/item/proc/can_be_equipped(mob/Mob, slot, disable_warning = FALSE)
```

Comment:
how do these relate to existing procs like mob/proc/mob_can_equip?

If they're intended as just an alternate code path, they should wrap the above

Reply:
As mentoined in description
> Current docs may not reflect the real procs! 

Because I would have to rewrite them with every PRs.
Welp, we can pass that part till last PRs, why not?